apps/platform/src/app/[chainId]/projects/propose/page.tsx

```diff
@@ -33,63 +29,91 @@ const getErrorMessage = (
 export default function CreateProjectPage() {
 	const t = useTranslations('proposalForm');
 
+	const {
+		selectedFiles,
+		handleFileChange,
+		openFilePicker,
+		removeFile,
+		inputRef,
+	} = useFileHandler();
+
 	const [isChecked, setIsChecked] = useState(false);
 	const [seconds, setSeconds] = useState(15);
+	const [isLoaded, setIsLoaded] = useState(false);
+	const [proposalFormData, setProposalFormData] = useState({
+		project: '',
+		location: '',
+		description: '',
+		name: '',
+		proposer: '',
+		email: '',
+		mobile: '',
+		isChecked: false,
+	});
 
 	const handleCheckboxChange = () => {
 		setIsChecked(prevState => !prevState);
+		setProposalFormData(prevData => ({
+			...prevData,
+			isChecked: !prevData.isChecked,
```

Comment:
I would move `isChecked` out of saved form data. It's something that has to be explicitly checked, and not a part of their proposal draft.

Reply:
Good suggstion, thx, removed